localization/english/wc_purge_l_english.yml

```diff
@@ -0,0 +1,41 @@
+﻿l_english:
+ purge.0010.t:0 "Today was a calm day"
+ purge.0010.desc:0 "We woke up early as always, didn't have much problem packing up everything and moving the camp to the next location. We barely saw a few deers on the way to OUR OBJECTIVE. It's almost too calm, like if something bad is going to happen..."
```

Comment:
OUR OBJECTIVE? Don't get.

Reply:
it's a placeholder, gonna use the purged barony's name.

Reply:
> it's a placeholder, gonna use the purged barony's name.

Better localize it now not to forget later.